applications/StructuralMechanicsApplication/tests/test_adjoint_loading_conditions.py

```diff
@@ -0,0 +1,132 @@
+from __future__ import print_function, absolute_import, division
+import KratosMultiphysics
+import KratosMultiphysics.StructuralMechanicsApplication as StructuralMechanicsApplication
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+import math
+
+class TestAdjointLoadingConditions(KratosUnittest.TestCase):
+
+    def __CheckSensitivityMatrix(self, sen_matrix, reference_sen_matrix, digits_to_check=5):
+        for i in range(sen_matrix.Size1()):
+            for j in range(sen_matrix.Size2()):
+                self.assertAlmostEqual(sen_matrix[i,j], reference_sen_matrix[i][j], digits_to_check)
+
+    def _SurfaceLoadCondition3D4N(self, prefix = ""):
+        current_model = KratosMultiphysics.Model()
+        mp = current_model.CreateModelPart("solid_part")
+        mp.AddNodalSolutionStepVariable(KratosMultiphysics.DISPLACEMENT)
+        mp.AddNodalSolutionStepVariable(KratosMultiphysics.REACTION)
+
+        #create nodes
+        mp.CreateNewNode(1,0.0,0.0,0.0)
+        mp.CreateNewNode(2,1.0,0.0,0.0)
+        mp.CreateNewNode(3,1.0,1.0,0.0)
+        mp.CreateNewNode(4,0.0,1.0,0.0)
+
+        #ensure that the property 1 is created
+        mp.GetProperties()[1]
```

Comment:
```suggestion
        props = mp.CreateNewProperties(1)
```
and then use it in the remaning code